src/StreamJsonRpc.Tests/JsonRpcWithFatalExceptionsTests.cs

```diff
@@ -170,6 +173,50 @@ public async Task CancelMayStillReturnErrorFromServer()
         Assert.False(this.serverRpc.IsDisposed);
     }
 
+    [Fact]
+    public async Task DisposedClientResultsInCancelled()
```

Comment:
This isn't related to server fatal exceptions. Can you move it to `JsonRpcTests.cs`?

Reply:
I took care of it.